### PR TITLE
gitea: Update to 1.16.3

### PR DESCRIPTION
**Pull request by xtkoba**

### CODE REVIEW

File: packages/gitea/build.sh [hunk 1]
```diff
@@ -22,6 +21,15 @@ termux_step_make() {
 	cp -a "$TERMUX_PKG_SRCDIR" "$GOPATH"/src/code.gitea.io/gitea
 	cd "$GOPATH"/src/code.gitea.io/gitea
 
+	go mod init || :
+	go mod tidy
+
+	for f in "$GOPATH"/pkg/mod/github.com/lib/pq@*/user_posix.go; do
+		chmod 0755 "$(dirname "$f")"
+		chmod 0644 "$f"
+		sed -i '/^\/\/ +build /s/ linux / linux,!android /g' "$f"
```
Comment (Grimler91) [inline packages/gitea/build.sh:30]: Could you add a comment about what the sed command does to make it easier to understand? 
My sed-knowledge is a bit weak, but I guess it replace `linux` with `linux,!android` if the line starts with `// +build`?

**Comment by Gusted** [comment]
FYI: [Gitea v1.16.3](https://github.com/go-gitea/gitea/releases/tag/v1.16.3) was released today might be nice to include that in this PR.

**Comment by techknowlogick** [comment]
Gitea 1.16.3 is now out, could you target that version instead?

**Comment by techknowlogick** [comment]
The TAGS env var also needs to be updated, you’ll need to add sqlite_unlock_notify in addition to the existing ones

**Comment by xtkoba** [comment]
Thanks!

**Comment by techknowlogick** [comment]
The changes look great! Thanks :)

**Pull request merged**